gcs/CONFIGURATION.md

```diff
@@ -138,6 +138,19 @@
     fs.gs.storage.http.headers.another-custom-header=another_custom_value
     ```
 
+*   `fs.gs.encryption.algorithm` (not set by default)
```

Comment:
Is there are possibility that these properties will be applicable to other non-CSEK encryption methods? If no, should we replace `encryption` with `csek` in the properties names? Or gsutil also uses `encryption` for configuration properties?

Reply:
yes, encryption_key is actually gsutil's name convention as well. I guess since there is no alternative encryption for CSEK so calling it encryption should be fine for now.